analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/Program.cs

```diff
@@ -38,7 +37,12 @@ public static void Main(string[] args)
         {
             if (args.Length == 1)
             {
-                var language = AnalyzerLanguage.Parse(args[0]);
+                var language = args[0] switch
+                {
+                    "cs" => AnalyzerLanguage.CSharp,
+                    "vbnet" => AnalyzerLanguage.VisualBasic,
+                    _ => throw new NotSupportedException("Unsupported argument: " + args[0]),
+                };
                 var root = new RuleDescriptorRoot(RuleDetailBuilder.GetAllRuleDetails(language).Select(RuleDetail.Convert));
                 Directory.CreateDirectory(language.ToString());
```

Comment:
I'd rather keep a hardcoded version for the directory name -  to keep the same directory names (e.g. `csharp` and `vbnet`)
- it's the same keys as rspec keys (not "yet another name to keep in mind")
- it doesn't have spaces inside (which don't work on Linux - I prefer no spaces in paths because of that)
- if in the unlikely even the Roslyn constants will change, we won't need to update the `pom.xml` because of that - it's all in our codebase
WDYT?

Reply:
and this logic would be strictly in this file - not in the `AnalyzerLanguage` which is common code with test infra